src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -0,0 +1,216 @@
+<Project>
+
+  <PropertyGroup>
+    <_MauiRuntimePackVersion>$(MauiVersion)</_MauiRuntimePackVersion>
+    <_MauiRuntimePackVersion Condition=" '$(_MauiRuntimePackVersion)' == '' ">**FromWorkload**</_MauiRuntimePackVersion>
+  </PropertyGroup>
+
+  <!-- Known framework references -->
+  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == '' ">
+    <KnownFrameworkReference
+        Include="Microsoft.Maui.Core"
+        TargetFramework="@TFM@"
+        RuntimeFrameworkName="Microsoft.Maui.Core"
+        DefaultRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        LatestRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        TargetingPackName="Microsoft.Maui.Core.Ref.any"
+        TargetingPackVersion="$(_MauiRuntimePackVersion)"
+        RuntimePackNamePatterns="Microsoft.Maui.Core.Runtime.**RID**"
+        RuntimePackRuntimeIdentifiers="any"
+        RuntimePackAlwaysCopyLocal="true"
+    />
+    <KnownFrameworkReference
+        Include="Microsoft.Maui.Controls"
+        TargetFramework="@TFM@"
+        RuntimeFrameworkName="Microsoft.Maui.Controls"
+        DefaultRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        LatestRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        TargetingPackName="Microsoft.Maui.Controls.Ref.any"
+        TargetingPackVersion="$(_MauiRuntimePackVersion)"
+        RuntimePackNamePatterns="Microsoft.Maui.Controls.Runtime.**RID**"
+        RuntimePackRuntimeIdentifiers="any"
+        RuntimePackAlwaysCopyLocal="true"
+    />
+    <KnownFrameworkReference
+        Include="Microsoft.Maui.Essentials"
+        TargetFramework="@TFM@"
+        RuntimeFrameworkName="Microsoft.Maui.Essentials"
+        DefaultRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        LatestRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        TargetingPackName="Microsoft.Maui.Essentials.Ref.any"
+        TargetingPackVersion="$(_MauiRuntimePackVersion)"
+        RuntimePackNamePatterns="Microsoft.Maui.Essentials.Runtime.**RID**"
+        RuntimePackRuntimeIdentifiers="any"
+        RuntimePackAlwaysCopyLocal="true"
+    />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == 'android' ">
+    <KnownFrameworkReference
+        Include="Microsoft.Maui.Core"
+        TargetFramework="@TFM@"
+        RuntimeFrameworkName="Microsoft.Maui.Core"
+        DefaultRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        LatestRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
+        TargetingPackName="Microsoft.Maui.Core.Ref.android"
+        TargetingPackVersion="$(_MauiRuntimePackVersion)"
+        RuntimePackNamePatterns="Microsoft.Maui.Core.Runtime.**RID**"
+        RuntimePackRuntimeIdentifiers="android"
+        Profile="Android"
+    />
```

Comment:
So I attempted to just define every platform, without conditions, but I get errors like:
```
C:\src\maui\bin\dotnet\sdk\6.0.300-preview.22168.1\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): error NETSDK1133: There was conflicting information about runtime packs available for Microsoft.Maui.Core:
Microsoft.Maui.Core.Runtime.**RID**
Microsoft.Maui.Core.Runtime.**RID**
Microsoft.Maui.Core.Runtime.**RID**
Microsoft.Maui.Core.Runtime.**RID**
```
Example log: [maui-pack-DEBUG.zip](https://github.com/dotnet/maui/files/8324845/maui-pack-DEBUG.zip)

I also tried making `%(TargetFramework)` have the profile in it, so `net6.0-android` instead of `net6.0`:
```
C:\src\maui\bin\dotnet\sdk\6.0.300-preview.22166.1\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(356,5): error NETSDK1073: The FrameworkReference 'Microsoft.Maui.Core' was not recognized
```
Example log: [msbuild.zip](https://github.com/dotnet/maui/files/8324860/msbuild.zip)

What is here now, is the same as we had before basically. Except I listed every platform to avoid using MSBuild properties.

Reply:
Let me check if I can just delete `%(Profile)`.